keras/layers/core.py

```diff
@@ -592,10 +592,21 @@ def antirectifier_output_shape(input_shape):
         model.add(Lambda(antirectifier,
                          output_shape=antirectifier_output_shape))
     ```
+    ```python
+        # add a layer that returns the hadamard product
+        # of the two input tensors
+
+        def hadamard_product(tensors):
+            return tensors[0]*tensors[1]
+
+        x1 = Dense(32, activation="relu", input_dim=3)(input_1)
+        x2 = Dense(32, activation="relu", input_dim=3)(input_2)
+        x = Lambda(hadamard_product)([x1, x2])
```

Comment:
can you also give the `output_shape` argument?

Reply:
since am trying to return a list of two tensors,  I am not sure how the output function should look like,
```python
def hadamard_product_sum_output_shape(input_shapes):
    shape1 = list(input_shapes[0])
    shape2 = list(input_shapes[1])
    assert input_shapes[0] == input_shapes[0] # else hadamard product isn't possible
    return tuple(shape1)
```
and
```
x, _ = Lambda(hadamard_product_sum, output_shape = hadamard_product_sum_output_shape)([x1, x2])
``` 
tried something like this, but failed. facing a list out of index error, probably because of the way I am passing the inputs for the shape. Can you help me out here?

Reply:
It should be return [tuple(shape1), tuple(shape2[:-1])]